VmAgent.Core/Interfaces/DockerContainerEngine.cs

```diff
@@ -417,7 +417,7 @@ private IList<string> GetVolumeBindings(SessionHostsStartInfo request, int sessi
                 {
                     string pathOnHost = request.UseReadOnlyAssets ? _vmConfiguration.GetAssetExtractionFolderPathForSessionHost(0, i) :
                     _vmConfiguration.GetAssetExtractionFolderPathForSessionHost(sessionHostInstance, i);
-
+                    _systemOperations.SetUnixOwnerIfNeeded(pathOnHost, true);
```

Comment:
can we add a comment on what's the purpose of that (run containers on user mode)